contracts/mocks/DummyDataFeed.sol

```diff
@@ -11,14 +11,17 @@ import '../interfaces/IDataFeed.sol';
 contract DummyDataFeed is IDataFeed {
     uint8 stateDecimals;
     int256 answer;
+    uint256 answeredAtOffset;
 
     /**
      * @param _decimals - Number of decimals the decimals function has to return
      * @param _answer - Value returned as the second argument of the return value of both getRoundData and latestRoundData
+     * @param _answeredAtOffset - Value deducted from the fourth argument of the return value of both getRoundData and latestRoundData which is the current block timestamp
```

Comment:
Im not sure if this is the right description. It is not just the offset?

Reply:
This just describes literally how the value is used in this mock.
The updatedAt parameter is always returned as the block.timestamp.
The updatedAtOffset provides a way to test the tolerance.